mds_loadtest/scripts/constants.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This newline padding disturbs me. Please remove :)

Reply:
removed